### ISSUE TITLE
Not able to install package

### ISSUE THREAD

**Issue by klipa-dev** [RyuuGan/sol-merger#28]
When I run 

`npm i --save-dev sol-merger`

I get error 

```
npm ERR! path /Users/path_to_folder/node_modules/sol-merger/dist/bin/sol-merger.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/path_to_folder/node_modules/sol-merger/dist/bin/sol-merger.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/my_username/.npm/_logs/2020-01-07T21_18_21_305Z-debug.log
```

**Comment by RyuuGan** [comment]
Hello @nklipa13, 

For some reason files for not published by to the recent version. So I have unpublished it. I'll publish new version as soon as posible.

Relates #19, #24 

Best regards,
Valerii Aligorskii

**Comment by RyuuGan** [comment]
Should be fixed in `v1.2.2`
